src/jit/compiler.h

```diff
@@ -1933,9 +1933,10 @@ class Compiler
     GenTreePtr gtNewIndexRef(var_types typ, GenTreePtr arrayOp, GenTreePtr indexOp);
 
     GenTreeArgList* gtNewArgList(GenTreePtr op);
-
     GenTreeArgList* gtNewArgList(GenTreePtr op1, GenTreePtr op2);
 
+    GenTreeArgList* gtNewAggregate(GenTree* element);
```

Comment:
What about `StructFieldList` or `StructFieldValues` (or even just `StructFields`)? Also, do you have any similar concept in the first class struct changes?